docs/references/database/list.md

```diff
@@ -0,0 +1 @@
+Get a list of all the user's databases. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's databases. [Learn more about different API modes](/docs/admin).
```

Comment:
>@[lohanidamodar](https://github.com/lohanidamodar) lohanidamodar [17 hours ago](https://github.com/appwrite/appwrite/pull/3391#discussion_r903170438)
We don't have query parameter is only for list documents, we don't yet have it here

>@[eldadfux](https://github.com/eldadfux) eldadfux [12 hours ago](https://github.com/appwrite/appwrite/pull/3391#discussion_r903275828)
Good catch. We should fix it here as well: https://appwrite.io/docs/server/functions?sdk=nodejs-default#functionsList

Functions will be in a separate PR